File: Coding/Python/leap_year.py
# Prints the next 20 leap years
import time

year = int(time.strftime("%Y"))+1
count = 0

while(count < 20):
    if (year%400==0):
        count+=1
        print year
    elif (year%4==0) and (year%100!=0):
        count+=1
        print year
    year+=1

File: Coding/Ruby/leap_year.rb
#Prints the years that are leap years
time = Time.new
year = time.year+1
count = 0

while(count<=20)
    if (year%400)==0
        puts "#{year} is a leap year\n"
        count+=1
    elsif (year%4)==0 && (year%100)!=0
        puts "#{year} is a leap year\n"
        count+=1
    end
    year+=1
end
